usecases/mail-sender/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+mail.from = noreply@notification.us-ashburn-1.oci.oraclecloud.com
+mail.fromName = OCI Notification
+mail.smtp.user = ocid1.user.oc1..aaaaaaaa3jfondpbrtjds64hk6jfzvtmgxqr7k4wni4t4jergmxe627hf2bq@ocid1.tenancy.oc1..aaaaaaaasu7rvefmsyk5kqczfmdqi5clpddejfjk2attdqnk6sbk72wajq5q.7l.com
+mail.smtp.password = 1#rMe.A(seqHm{vF[Jkt
```

Comment:
Credentials shouldn't be posted to the git repo.  Please rotate these credentials (assuming they're actual) and use a placeholder value (at minimum) or better yet reference an environment variable (so that no credentials are hard-coded in files).

Reply:
Fixed